src/components/form-attachment/list.vue

```diff
@@ -357,6 +386,30 @@ export default {
         });
       this.plannedUploads = [];
       this.unmatchedFiles = [];
+    },
+    fetchDatasets() {
+      this.datasets.request({
+        url: apiPaths.datasets(this.projectId),
+        resend: false
+      }).catch(noop);
+    },
+    showRestoreModal(name) {
+      this.restoreModal.attachmentName = name;
+      this.showModal('restoreModal');
+    },
+    afterRestore() {
+      this.alert.success(this.$t('alert.restore', {
+        attachmentName: this.restoreModal.attachmentName
+      }));
+
+      this.attachments.patch(() => {
```

Comment:
Nice use of `patch()`! This will group these changes together within Vue Devtools.